src/components/RequestGroupComposition/CustomField.vue

```diff
@@ -23,9 +23,11 @@
           :value="value"
           :state="validationState"
           :type="type"
+          :list="field + '-field-' + $parent.id + '-list'"
```

Comment:
The changes in this file are unrelated to the extra params fields, its just something I had locally to use in hermes UI at some point. It adds support for auto-completion lists for string input fields basically.